File: BookEcommerce/Dto/Mail.cs


using System.Net.Mail;

namespace BookStore.Dto;
public class MailRequest
{
    public string ToEmail { get; set; } = null!;
    public string Subject { get; set; }
    public string Body { get; set; }
    public MailPriority Priority { get; set; } = MailPriority.Normal;
}
public class MailSetting
{
    public string From { get; set; } = null!;
    public string DisplayName { get; set; } = null!;
    public string Host { get; set; } = null!;
    public int Port { get; set; }
    public string Password { get; set; } = null!;
}

File: apps/admin/src/types/temp/mail.ts
import { MailPriority } from './mailPriority';

export interface MailRequest {
    toEmail: string;
    subject: string;
    body: string;
    priority: MailPriority;
}

export interface MailSetting {
    from: string;
    displayName: string;
    host: string;
    port: number;
    password: string;
}